config/env/crystals/lattice_parameters.yaml

```diff
@@ -0,0 +1,28 @@
+defaults:
+  - base
+
+_target_: gflownet.envs.crystals.lattice_parameters.LatticeParameters
+
+id: lattice_parameters
+
+lattice_system: triclinic
+min_length: 1.0
+max_length: 5.0
```

Comment:
Could you please remind me if these default values were based on something? On a data set?

Reply:
That's a good question... I seem to recall asking Divya at some point for the stats from dataset, but I just compared this against the triclinic CSV used in this PR, and it doesn't fit within this range.

I suspect it has to do with the file being generated prior to migration to conventional, but we are trying to figure this out right now.

Reply:
Have you converged into something about this question?

Reply:
Yes, https://github.com/alexhernandezgarcia/gflownet/pull/217, discussion on Slack. KDE needs to be refitted.